world-server/src/network/Server.ts

```diff
@@ -1,6 +1,6 @@
 import * as http from 'http';
 import * as websocket from 'websocket';
-import { Logger, Messages, Router } from 'world-core';
+import { Logger, Messages, Router } from '../../../world-core';
```

Comment:
This is not OK, you should always import from `world-core` and expose the types there. Did you forget to `yarn core build`?

Reply:
Even after `yarn core build` the error persists. Any suggestions?